enhancements/olm-toggle-copied-csvs.md

```diff
@@ -48,7 +48,7 @@ When an Operator is installed by OLM, a stripped down copy -- e.g. a stub -- of
 
 ## Proposal
 
-Provide a CRD for OLM that allows users to enable and disable Copied CSVs for CSVs installed in `AllNamespace` mode, then for each non-Copied CSV, a single [Operator](https://github.com/operator-framework/enhancements/blob/master/enhancements/simplify-olm-apis.md#proposal) resource will be generated. This cluster-scoped resource will take the place of any related Copied CSVs.
+Provide a CRD for OLM that allows users to enable and disable Copied CSVs for CSVs installed in `AllNamespace` mode.
```

Comment:
Can we bump the `last-updated` and `status` field as well now that these changes have landed in the master branch?

Reply:
Done!